awscfncli2/cli/utils/deco.py

```diff
@@ -29,3 +30,35 @@ def wrapper(ctx, *args, **kwargs):
             raise click.Abort
 
     return wrapper
+
+
+def retry_on_throttling(tries=4, delay=2, backoff=2):
+    """
```

Comment:
considering use a 3rd party backup off library with jitter support?

Reply:
Reimplemented retry mechanism using [backoff](https://github.com/litl/backoff) library that implements jitter algorithm